stacks-signer/src/main.rs

```diff
@@ -294,6 +296,57 @@ fn handle_generate_files(args: GenerateFilesArgs) {
     );
 }
 
+fn handle_ping(args: PingArgs) {
+    debug!("Pinging Signers...");
+    let config = Config::try_from(&args.config).unwrap();
+
+    // Ping::RunLoop doesnt use the command channel.
+    let (_cmd_send, cmd_recv) = channel();
+    let (res_send, res_recv) = channel();
+    let ev = StackerDBEventReceiver::new(vec![config.stackerdb_contract_id.clone()]);
+    //let stop_signal = ev.get_stop_signaler().unwrap();
+
+    let mut runloop = PingLoop::from(&config);
+
+    let ping = Ping::new();
+    let msg: LatencyPacket = ping.into();
+
+    // Prime the stacks-client with the user-suggested slot_version.
+    runloop.increase_slot_version_counter(
+        config.signer_id,
+        &msg,
+        args.slot_version.saturating_sub(1),
+    );
+    let running_signer = Signer::new(runloop, ev, cmd_recv, res_send)
+        .spawn(config.endpoint)
+        .unwrap();
+
+    // hacky, single use-client to send the request.
+    let chunk_ack = {
+        let mut client = StacksClient::from(&config);
+
+        // `send_message_with_retry` will increment the counter prior use; POLA, use the value provided by user.
```

Comment:
What's `POLA`?

Reply:
Ack. Principle of least astonishment. It was a reminder to myself why I subtract one from the slot_version.